src/modules/dream/geometry/Config.h

```diff
@@ -24,8 +24,8 @@ class Config {
   static constexpr int MaxRing{11};
   static constexpr int MaxFEN{11};
 
-  enum DetectorInstance { NONE, DREAM, MAGIC };
-  enum ModuleType { BwEndCap, FwEndCap, Mantle, HR, SANS, FR, PA };
+  enum DetectorInstance { NONE, DREAM, MAGIC, HEIMDAL };
+  enum ModuleType { BwEndCap, FwEndCap, Mantle, HR, SANS, FR, PA, HeimdalMantle};
```

Comment:
IF there are multiple Mantle, like dream and heimdal mantle. Maybe would be better to call the dream one as DreamMantle and not just mantle to be precise on that. I know everything started with dream but later this can cause confusion for those hows not have the history of the code.

Reply:
Dream and Magic have identical interpretation. I've renamed Mantle to DreamMantle and 
mention in the doxygen header that this is also used by Magic.

Is this acceptable?

Reply:
Yes, that's good if it's the same object.